File: 1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = lines.shift().split(' ');

var A = parseInt(a[0]);
var B = parseInt(a[1]);
var C = parseInt(a[2]);

if (A > B && A > C) {
	console.log(A + ' eh o maior');
} else if (B > C) {
	console.log(B + ' eh o maior');
} else {
	console.log(C + ' eh o maior');
}

File: 1013.py
a = input().split()

A = int(a[0])
B = int(a[1])
C = int(a[2])

MaiorAB = (A+B+abs(A-B))/2

MaiorTotal = (C+MaiorAB+abs(C-MaiorAB))/2

print('{:.0f} eh o maior'.format(MaiorTotal))